### PR TITLE
Updated broken link in README for notebook demo

### PR DESCRIPTION
**Pull request by VaishnaviNandakumar**
### Pull Request Description
- Small update in ReadMe for the broken link in Notebook Demo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ Featuretools contains many [different types of built-in primitives](https://prim
 ## Demos
 **Predict Next Purchase**
 
-[Repository](https://github.com/Featuretools/predict_next_purchase/) | [Notebook](https://github.com/Featuretools/predict_next_purchase/blob/main/Tutorial.ipynb)
+[Repository](https://github.com/Featuretools/predict_next_purchase/) | [Notebook](https://github.com/alteryx/open_source_demos/blob/main/predict-next-purchase/Tutorial.ipynb)
```
Comment (gsheni) [inline README.md:84]:
```suggestion
[Repository](https://github.com/alteryx/open_source_demos/blob/main/predict-next-purchase/) | [Notebook](https://github.com/alteryx/open_source_demos/blob/main/predict-next-purchase/Tutorial.ipynb)
```

**Comment by gsheni** [comment]
@VaishnaviNandakumar Thanks for the contribution. I had one suggestion above. Also you should update the release notes
- https://github.com/alteryx/featuretools/blob/main/docs/source/release_notes.rst

**Review by gsheni — DISMISSED** [review]

**Review by gsheni — APPROVED** [review]

**Comment by gsheni** [comment]
Congrats on your first merge request, @VaishnaviNandakumar !

**Comment by VaishnaviNandakumar** [comment]
@gsheni Thank you!

**Pull request merged**